zlib.scm

```diff
@@ -136,11 +139,16 @@
            (unless eof? ; free up memory
              (inflate-end stream)))))))
 
-(define deflate-init (foreign-lambda int "deflateInit" z-stream int))
+(define deflate-init (foreign-lambda int "deflateInit2" z-stream int int int int int))
 (define deflate (foreign-lambda int "deflate" z-stream int))
 (define deflate-end (foreign-lambda void "deflateEnd" z-stream))
 
-(define (open-zlib-compressed-output-port #!optional (port (current-output-port)))
+(define (open-zlib-compressed-output-port #!optional (port (current-output-port))
+                                          #!key (level -1)
+                                                (method Z_DEFLATED)
+                                                (window-bits 15)
```

Comment:
Python has constants here e.g `MAX_WBITS` but idk if this is actually that useful

Reply:
It is as the manipulation of `MAX_WBITS` allows selecting a different format to compress/decompress. Therefore I'd export it as `wbits/max` or something.

Reply:
I've extended the test suite to make use of this for handling other formats as well. I'll submit a PR once this one is merged.

Reply:
Done, and thanks for the clarification